### ISSUE TITLE
Unable to run the example | Docker issue even when installed and running.

### ISSUE THREAD

**Issue by shivendrasoni** [langchain-ai/langgraph-studio#10]
<img width="775" alt="Screenshot 2024-08-02 at 3 24 35 AM" src="https://github.com/user-attachments/assets/56468799-a982-4fc5-9a02-0e5c36afd95f">

I keep getting this error, when running example or my own graph. 

I have docker installed and running.
```
Docker Engine
v27.1.1
```

I have added it to the path as well.

**Comment by dqbd** [comment]
Hello @shivendrasoni! Sorry to hear that, could you please try running this command in terminal and paste the output here?

```
dscl . -read ~/ UserShell | sed 's/UserShell: //'
```

**Comment by shivendrasoni** [comment]
This is what I got: 
`<dscl_cmd> DS Error: -14136 (eDSRecordNotFound)`

**Comment by dqbd** [comment]
Hello! Published a new 0.0.11 version of the app, which should hopefully resolve the issue. Will close the issue for now, feel free to reopen if the issue is still not resolved.
